src/lib/server/database.ts

```diff
@@ -85,5 +85,5 @@ client.on("open", () => {
 	assistants.createIndex({ userCount: 1 }).catch(console.error);
 	assistants.createIndex({ featured: 1, userCount: -1 }).catch(console.error);
 	assistants.createIndex({ modelId: 1, userCount: -1 }).catch(console.error);
-	reports.createIndex({ assistantId: 1 }).catch(console.error);
+	reports.createIndex({ createdBy: 1, assistantId: 1 }).catch(console.error);
```

Comment:
I think it's nice to keep both indexes - that way we can easily see how many times a particular assistant has been reported, or which assistant is the most reported

Reply:
wdyt 2750cbcb322e04cd1d85f5fc969c6af486e7e976 ?

Reply:
nice!